contracts/native/evm/evminternal/management.go

```diff
@@ -29,18 +38,71 @@ var ManagementHandlers = []coreutil.ProcessorEntryPoint{
 	evm.FuncWithdrawGasFees.WithHandler(withdrawGasFees),
 	evm.FuncGetOwner.WithHandler(getOwner),
 	evm.FuncGetGasPerIota.WithHandler(getGasPerIota),
+	evm.FuncSetBlockTime.WithHandler(setBlockTime),
+}
+
+func EVMStateSubrealm(state kv.KVStore) kv.KVStore {
+	return subrealm.New(state, keyEVMState)
 }
 
 func InitializeManagement(ctx iscp.Sandbox) {
 	ctx.State().Set(keyGasPerIota, codec.EncodeUint64(evm.DefaultGasPerIota))
 	ctx.State().Set(keyEVMOwner, codec.EncodeAgentID(ctx.ContractCreator()))
 }
 
-func requireOwner(ctx iscp.Sandbox) {
+func setBlockTime(ctx iscp.Sandbox) (dict.Dict, error) {
+	params := kvdecoder.New(ctx.Params(), ctx.Log())
```

Comment:
missing `requireOwner`?
```suggestion
	requireOwner(ctx)
	params := kvdecoder.New(ctx.Params(), ctx.Log())
```

Reply:
Nice catch!